WebDriverAgentMac/WebDriverAgentLib/Utilities/FBW3CActionsSynthesizer.m

```diff
@@ -792,14 +792,6 @@ @implementation FBW3CActionsSynthesizer
   });
 
   NSArray<NSDictionary<NSString *, id> *> *actionItems = [actionDescription objectForKey:FB_KEY_ACTIONS];
-  if (nil == actionItems || 0 == actionItems.count) {
```

Comment:
should we short circuit here if this condition is true?

Reply:
Sure, let me keep them in case we use them in another place without guard in this PR